stixcore/io/fits/processors.py

```diff
@@ -107,6 +107,18 @@ def generate_filename(cls, product, *, version, date_range, status='', header=Tr
     @classmethod
     def generate_common_header(cls, filename, product, *, version=0):
 
+        user_req = ''
+        if 'request_id' in product.control.colnames:
+            rid_entry = np.atleast_1d(product.control['request_id'][0])
+            if len(rid_entry) > 1:
+                user_req = f'{rid_entry[1]:010d}-{rid_entry[0]:05d}'
```

Comment:
Can/does this every happen what does `f'{rid_entry[1]:010d}-{rid_entry[0]:05d}'` look like?

Reply:
yes this will happen in the cases where the rid is composed with RID and TC-SeqeunceID 

`solo_LB_stix-21-6-20_0000000000-9999999999_V01_0079424512-50488.fits`

in the `request_id` column we have an array in this case:

`array([   50488, 79424512])`

the entry value in the header will then also be: `0079424512-50488`